widget/bat.lua

```diff
@@ -22,7 +22,7 @@ local function factory(args)
     local pspath = args.pspath or "/sys/class/power_supply/"
 
     if not fs.is_dir(pspath) then
-        naughty.notify { text = "lain.widget.bat: invalid power supply path", timeout = 0 }
+        naughty.notify { text = tostring(...):match(".*lain") .. ".widget.bat: invalid power supply path", timeout = 0 }
```

Comment:
`naughty.notify` is not vararg, so this raises the following error:

```lua
cannot use '...' outside a vararg function near '...'
```

Since we cannot touch `naughty.notify`, please revert this change.